Sources/ExyteChat/ChatView/ChatView.swift

```diff
@@ -184,19 +188,61 @@ public struct ChatView<MessageContent: View, InputViewContent: View, MenuAction:
                     .ignoresSafeArea()
                 }
             }
-
+            .onAppear() {
+              if isGiphyAvailable() {
+                if let giphyKey = giphyConfig.giphyKey {
+                  if !giphyConfigured {
+                    giphyConfigured = true
+                    Giphy.configure(apiKey: giphyKey)
```

Comment:
this line does the auth with the giphy api, you need to have been given an api key from the sdk team first